charts/argocd-apps/values-test.yaml

```diff
@@ -295,3 +295,100 @@ applications:
         value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
       - name: GOVUK_APP_NAME
         value: collections
+- name: whitehall-admin
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/whitehall
+      tag: f01a5161aa0d93b2efdd0775c61081695f12447a  # To be edited by automation (not yet implemented).
```

Comment:
I think the format is `# TODO: <message>` for automation/search to find it easier

Reply:
That right, thanks 
i will update this with an amend